java/src/main/java/org/eclipse/ditto/client/internal/DefaultDittoClient.java

```diff
@@ -79,7 +79,7 @@
     private static final String SELECTOR_INCOMING_MESSAGE = "incoming-message";
 
     private static final String THING_PATTERN = "/things/{0}";
-    private static final String ACL_PATTERN = THING_PATTERN + "/acl/{1}";
+    @Deprecated private static final String ACL_PATTERN = THING_PATTERN + "/acl/{1}";
```

Comment:
I don't think we need to deprecate internal implementation. It shouldn't be called directly and we will remove it eventually.